cartography/data/jobs/analysis/aws_apigateway_asset_exposure.json

```diff
@@ -10,13 +10,49 @@
 			"iterative": true,
 			"iterationsize": 100
 		},
+		{
+			"query": "MATCH (rest_api_stage:APIGatewayStage)<-[:ASSOCIATED_WITH]-(:APIGatewayRestAPI)<-[:RESOURCE]-(:AWSAccount{id: $AWS_ID})<-[:OWNER]-(:CloudanixWorkspace{id: $WORKSPACE_ID}) WHERE EXISTS(rest_api_stage.exposed_internet) WITH rest_api_stage LIMIT $LIMIT_SIZE REMOVE rest_api_stage.exposed_internet, rest_api_stage.exposed_internet_type return COUNT(*) as TotalCompleted",
```

Comment:
public facing for stage does not make a lot of sense. because, it's more of a way to identify an environment. let's skip this object.